app/Http/Controllers/LoginController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\LoginRequest;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Redirect;
+
+class LoginController extends Controller
+{
+    public function index()
+    {
+        if (Auth::check()) {
+            return Redirect::route('home');
+        }
+
+        return view('login');
+    }
+
+    public function login(LoginRequest $request)
+    {
+        $login = [
+            'email' => $request->email,
+            'password' => $request->password,
+        ];
+
+        if (Auth::attempt($login, isset($request->remember))) {
+            return Redirect::route('home');
+        }
+
+        return view('login', ['err' => 'Email hoặc Password không chính xác']);
```

Comment:
```suggestion
        return view('login')
            ->withErrors([
                'credentials' => __('auth.invalid'),
            ]);
```
M suggest sử dụng withErrors để trả về thông báo lỗi nha.